src/cromshell/status/command.py

```diff
@@ -114,34 +115,74 @@ def main(config, workflow_id):
     return ret_val
 
 
-def get_metadata_status_summary(workflow_metadata):
+def get_all_call_values(nested_dictionary):
+    """Summarizes a given workflow metadata dictionary into a
+    new dictionary with only task name and status"""
+    temp_dic1 = {}
```

Comment:
`temp_dic1/2/3` could use better names to reflect what their intended use/contents are.

Reply:
done